tests/components/aussie_broadband/test_config_flow.py

```diff
@@ -229,7 +186,7 @@ async def test_reauth(hass: HomeAssistant) -> None:
         context={"source": config_entries.SOURCE_REAUTH},
         data=FAKE_DATA,
     )
-    assert result5["step_id"] == "reauth"
+    assert result5["step_id"] == "reauth_confirm"
 
     with patch("aussiebb.asyncio.AussieBB.__init__", return_value=None), patch(
```

Comment:
This still looks weird, but we can address that in a separate PR.

Reply:
Yeah I expected we may need to discuss this further. I tried removing the __init__ patch and things broke. Now that the library has been bumped, I think I will be able to do a PR focusing on tests.

Reply:
It should be enough to patch `"homeassistant.components.aussie_broadband.config_flow.AussieBB.login"` and the same for `"get_services"`.